virtualizarr/tests/test_backend.py

```diff
@@ -330,15 +331,21 @@ def test_open_empty_group(empty_netcdf4_file):
 class TestOpenVirtualDatasetHDFGroup:
```

Comment:
These tests look at functionality that doesn't exist in Sean's backend

Reply:
Really? I swear opening specific groups is supported https://github.com/zarr-developers/VirtualiZarr/blob/2c856b188365bda3e1810ab96d5d3b2260aaa625/virtualizarr/readers/hdf/hdf.py#L364

Reply:
I was moving too quick. You are right, it was only the one test for nested groups that was unimplemented. The rest were failing for different fixable reasons